renovate.json

```diff
@@ -17,6 +17,7 @@
   "semanticCommitScope": "",
   "semanticCommitType": "build",
   "separateMajorMinor": false,
+  "ignoreDeps": ["@angular/bazel"],
```

Comment:
We are not updating angular Bazel automatically, so this is unnecessary I think

Reply:
You are right, I missed https://github.com/angular/components/blob/c14aadd880a5afca92d7da11e3c65cb78d491e1b/renovate.json#L22-L25